### PR TITLE
refactor: changed release name and namespaces for all environments

### PR DESCRIPTION
**Pull request by joaocosta-codacy**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -432,15 +436,18 @@ workflows:
           requires:
             - deploy_to_doks
 
-  hourly_pipeline:
+  staging_pipeline:
```
Comment (paulopontesm) [inline .circleci/config.yml]: put this in a separate pr

Comment (joaocosta-codacy) [reply .circleci/config.yml]: I have reverted this stuff and will return to it after we have the codacy-website publishing to dev sh working fine. 
It makes more sense to talk about the staging environment then.

File: .circleci/config.yml [hunk 2]
```diff
@@ -438,9 +442,9 @@ workflows:
           dev_branch: "master"
           release_branch: "release"
           filters:
-              branches:
-                only:
-                  - feature/hourly-tests
+            branches:
+              only:
+                - feature/hourly-tests
```
Comment (paulopontesm) [inline .circleci/config.yml]: can we just delete this? if you are changing

File: .doks/values.yaml [hunk 3]
```diff
@@ -9,8 +9,13 @@ postgres:
     size: 1000Gi
 
 portal:
+  selfhosted: true
```
Comment (paulopontesm) [inline .doks/values.yaml]: what??? what is this?

File: .doks/values.yaml [hunk 4]
```diff
@@ -9,8 +9,13 @@ postgres:
...
   replicaCount: 2
 
+portal:
+  dev: true
```
Comment (paulopontesm) [inline .doks/values.yaml]: another portal?

File: codacy/requirements.yaml [hunk 5]
```diff
@@ -25,11 +25,18 @@ dependencies:
...
+  repository: https://charts.codacy.com/unstable
+  git: git@bitbucket.org:qamine/portal.git
+  condition: dev.selfhosted
+
 - name: portal
   version: ">=5.0.0"
   repository: https://charts.codacy.com/stable
   git: git@bitbucket.org:qamine/portal.git
 
+
```
Comment (paulopontesm) [inline codacy/requirements.yaml]: do we need this new line?

File: .doks/values.yaml (hunk 3 above)
Comment (joaocosta-codacy) [reply .doks/values.yaml]: accidental commit. removed.
